lib/Fetcher/FeedFetcher.php

```diff
@@ -135,7 +136,19 @@ public function fetch(
             $lastModified = null;
         }
         $url = $url2->getNormalizedURL();
-        $resource = $this->reader->read($url, null, $lastModified);
+
+        try {
+            $resource = $this->reader->read($url, null, $lastModified);
+        } catch (NoAccurateParserException) {
+            $this->logger->warning(
+                'Feed {url} cannot be be parsed',
+                [
+                'url'   => $url,
+                ]
+            );
+            throw new ReadErrorException("Feed cannot be parsed.");
```

Comment:
Why are we throwing this?

Reply:
From the summary: "Throwing a ReadErrorException because that is how we track feed fetching errors in the service."

Reply:
Maybe we can just add the NoAccurateParserException to that catch block?

Reply:
Maybe yes, I will look at it again when I have time.